datafusion/core/src/physical_planner.rs

```diff
@@ -562,8 +562,7 @@ impl DefaultPhysicalPlanner {
                     // doesn't know (nor should care) how the relation was
                     // referred to in the query
                     let filters = unnormalize_cols(filters.iter().cloned());
-                    let unaliased: Vec<Expr> = filters.into_iter().map(unalias).collect();
```

Comment:
It's useless

Reply:
I think we have to rely on our test coverage here, and given all the existing tests pass I agree!

Reply:
Yes, theoretically and in terms of code, the alias in the filter expression does not affect the schema or output.